meta-mentor-staging/recipes-core/volatile-binds/volatile-binds.bbappend

```diff
@@ -0,0 +1,5 @@
+# mount /etc/systemd with rw rights.
+BIND_DIR += "\
+    /var/volatile/systemd /etc/systemd\n\
+"
+VOLATILE_BINDS += "${@bb.utils.contains('IMAGE_FEATURES', 'read-only-rootfs', '${BIND_DIR}', '', d)}"
```

Comment:
IMAGE_FEATURES is image level by definition, it can be set in the image recipe itself, not just globally, so you can't access that here. Also, volatile-binds is only installed when doing read only builds, so this is just redundant. Drop the conditional entirely, or better yet, replace it with a conditional on the system distro feature.

Reply:
I have dropped this changes. Kindly review again.